--- Release/libs/websocketpp/websocketpp/transport/asio/security/tls.hpp.orig	2017-05-08 10:26:46.716716379 +0200
+++ Release/libs/websocketpp/websocketpp/transport/asio/security/tls.hpp	2017-05-08 12:19:43.082078616 +0200
@@ -307,13 +307,9 @@
      */
     lib::error_code translate_ec(boost::system::error_code ec) {
         if (ec.category() == boost::asio::error::get_ssl_category()) {
-            if (ERR_GET_REASON(ec.value()) == SSL_R_SHORT_READ) {
-                return make_error_code(transport::error::tls_short_read);
-            } else {
                 // We know it is a TLS related error, but otherwise don't know
                 // more. Pass through as TLS generic.
                 return make_error_code(transport::error::tls_error);
-            }
         } else {
             // We don't know any more information about this error so pass
             // through
